### PR TITLE
Refactor length check for thing type path.

### PR DESCRIPTION
**Pull request by dairyo**
According to our discussion on [these comments](https://github.com/KiiPlatform/thing-if-ThingSDK/pull/101/files#r116163379), I refactored length check procedure.

What do you think about this style? If you like this style, I will refactor other length checkings in this PR.

Related PR: #101

### CODE REVIEW

**Review by satoshikumano — COMMENTED** [review]
Looks Good!

**Comment by dairyo** [comment]
@satoshikumano Fixed all resource path creation procedure.
Could you review this?

File: kii_thing_if.c [hunk 1]
```diff
@@ -825,21 +876,23 @@ static int prv_kii_thing_if_get_anonymous_token(
 {
     char resource_path[64];
     kii_json_field_t fields[2];
+    size_t oauth_path_length =
+        strlen(APP_PATH) +
```
Comment (satoshikumano) [inline kii_thing_if.c:881]: No need to use CONST_STRLEN ?

**Review by satoshikumano — COMMENTED** [review]
Left comment.

**Comment by dairyo** [comment]
@satoshikumano Fixed according to our voice discussion.

We check succeeded of function with local variable.

**Pull request merged**